lib/flappi/optional/flappi.rake

```diff
@@ -13,7 +13,7 @@ namespace :flappi do
 
       target_paths.each do |document_version, to_path|
         paths = Flappi.configuration.definition_paths[document_version]
-        paths = paths.is_a?(Array) ? paths : [paths]
+        paths = [paths] unless paths.is_a?(Array)
```

Comment:
I would have thought that

```ruby
paths = Array(paths)
```

does exactly that. 🤔

Reply:
Yes, what you have works too and it's more succinct! Updated in [763e16c](https://github.com/sharesight/flappi/pull/63/commits/763e16c33e8ab94187ab86d4b197e7ed9b38d2a7)—thanks!